guizero/Waffle.py

```diff
@@ -142,14 +192,6 @@ def enabled(self):
     @enabled.setter
     def enabled(self, value):
         self._enabled = value
-    
-    def disable(self):
```

Comment:
enabled \ disabled should be implemented in Waffle as there implementation is specific to Waffle and not shared with any other component.